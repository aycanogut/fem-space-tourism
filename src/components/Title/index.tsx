import React, { FC } from 'react'

import { ITitleProps } from '../../interfaces/Title.interface'

const Title: FC<ITitleProps> = ({ id, title }) => {
  return (
    <header className="flex w-full justify-center gap-4 pt-[85px] font-barlow text-16 font-normal uppercase tracking-wider sm:justify-start sm:pt-[133px] sm:text-20 md:tracking-widest lg:pt-[200px] lg:text-28 lg:tracking-widester">
      <span className=" font-bold text-white/[.5]">{id}</span>
      <h1 className="text-white">{title}</h1>
    </header>
  )
}

export default Title
