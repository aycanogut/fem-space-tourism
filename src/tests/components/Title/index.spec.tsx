import renderer from 'react-test-renderer'
import Title from '../../../components/Title'

it('renders correctly', () => {
  const tree = renderer
    .create(<Title id="01" title="PICK YOUR DESTINATION" />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
