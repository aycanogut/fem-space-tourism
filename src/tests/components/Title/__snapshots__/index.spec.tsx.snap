// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="flex w-full justify-center gap-4 pt-[85px] font-barlow text-16 font-normal uppercase tracking-wider sm:justify-start sm:pt-[133px] sm:text-20 md:tracking-widest lg:pt-[200px] lg:text-28 lg:tracking-widester"
>
  <span
    className=" font-bold text-white/[.5]"
  >
    01
  </span>
  <h1
    className="text-white"
  >
    PICK YOUR DESTINATION
  </h1>
</header>
`;
